archives/2020.md

```diff
@@ -27,6 +27,7 @@
 
 | Conference | Location | Date | Submission Deadline | Flight / Hotel? | CoC? |
 | ---------- | -------- | ---- | ------------------- | ------------------------ | ---- |
+| [render(ATL)](https://www.renderatl.com/) | Atlanta, GA | May 4-6th | TBD | Yes | [Yes](https://www.renderatl.com/code-of-conduct) |
```

Comment:
@js1599 is there a talk submission deadline?

Reply:
Whoa super cool conference!!

I'm not seeing a CFP link anywhere. Have all the speakers already been chosen?

Reply:
Good Morning! Sorry for the delay! 

As of now, all speakers have been chosen, but we will hold a CFP for 1 more local Georgian speaker to be added onto the list. In addition, we are allowing anyone with enough courage to grace our secondary stage, that'll also be broadcasted live into the main event room (main stage). We're doing this because we recognize not everyone has had the privilege of knowing how to do a CFP, or even where to look to apply. So they'll be allowed onto the stage to just "say what u wana say" with a room of 50ish but broadcasted to everyone at the event. Makes the introduction to speaking much easier on newbies

Reply:
Hey ya'll. Could we happen to get this merged into master this week? Thanks again